docs/components/DocumentationPageLayout.js

```diff
@@ -1,56 +0,0 @@
-import styled, { keyframes, css } from 'react-emotion';
```

Comment:
Was this one never used? 😮

Reply:
Yes, I was surprised too. Triple checked for usages. Seems like it was replaced by `DocumentationNestedScrollLayout`